packages/desktop-client/src/components/transactions/MobileTransaction.js

```diff
@@ -934,7 +934,6 @@ class Transaction extends PureComponent {
           backgroundColor: 'white',
           border: 'none',
           width: '100%',
-          '&:active': { opacity: 0.1 },
```

Comment:
This was kinda annoying on mobile devices.. when you hold the transaction - it change its style. When you release - it opens the transaction (as  expected). It would be nicer to not have this weird in-between style.